README.md

```diff
@@ -38,109 +56,125 @@ No faster way to get started than by diving in and playing around with a demo.
 
 Need quickstarts to begin your Redis AI journey? **Start here.**
 
-### Non-Python Redis AI Recipes
-
-#### ☕️ Java
+### ☕️ Java AI Recipes
```

Comment:
Seems odd that this is first since we don't mention anywhere that all other recipes are python. I think it reads like JAVA is our main language here